lib/rzo/app/templates/environment.conf.erb

```diff
@@ -0,0 +1,18 @@
+# Each environment can have an environment.conf file. Its settings will only
```

Comment:
I'm actually going to remove this template, the environment.conf is inlined in the generated Vagrantfile.  For example:

```ruby
config.vm.provision 'shell', inline: "echo 'modulepath = ./modules:./puppetdata/modules:./ghoneycutt/modules' > /tmp/bootstrap_puppet4/environment.conf"
```

This was a lot easier than managing different environment files for different VM types.